UTorrentClientApi/Data/Props.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+namespace UTorrent.Api.Data
+{
+   public class Props
+    {
+        public string hash { get; set; }
```

Comment:
Can you use Pascal case for property name?

Reply:
No problems, i am more used to camelCase